sbin/Makefile.am

```diff
@@ -55,12 +28,29 @@ if ENABLE_VNETBUILD
 scripts += vnetbuild
 endif
 
-sbin_SCRIPTS = $(scripts)
+CLEANFILES = install.config
 
-EXTRA_DIST = \
-	commands.sed.in \
-	$(scriptsin) \
-	$(inclib_DATA)
+inclib_DATA = \
+	functions.common \
+	install.config \
+	$(NULL)
+
+inclib_SCRIPTS = $(scripts)
 
-uninstall-local:
+EXTRA_DIST = \
+	functions.common \
+	install.config.in \
+	$(scripts) \
+	$(NULL)
+
+install-exec-hook:
+	$(MKDIR_P) $(DESTDIR)$(sbindir)
+	for i in $(scripts); do \
+		$(RM) -f $(DESTDIR)$(sbindir)/$$i; \
+		$(LN_S) $(DESTDIR)$(inclibdir)/$$i $(DESTDIR)$(sbindir); done
+
+uninstall-hook:
+	for i in $(scripts); do \
+		$(RM) -f $(DESTDIR)$(sbindir)/$$i; done
 	@-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(inclibdir)
+	@-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(sbindir)
```

Comment:
automake standard is not to remove empty directories, packager can do this if it likes. this is true to any directory created.